### PR TITLE
New biome, plus new bush type.

### PR DESCRIPTION
**Pull request by Vulpicula**
Was hoping to have more done, but uh, here goes.

Biome itself isn't too special, just a redux of the temperate small mountains biome to add more variation, as otherwise there'd just be that one type.

Might tweak it some more in the future, and I'd encourage anyone else to also.

### CODE REVIEW

**Comment by Sancires** [comment]
Looks good, will review more in depth tonight

File: biomes/land/mountains-small/temperate/semi-humid/highlands.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+id: HIGHLANDS
+type: BIOME
+extends: [ EQ_ERODED_TERRACED_MOUNTAINS, CARVING_LAND, BASE ]
+vanilla: minecraft:plains
```
Comment (Sancires) [inline biomes/land/mountains-small/temperate/semi-humid/highlands.yml:4]: Should _probably_ not use plains: for future reference.

File: biomes/land/mountains-small/temperate/semi-humid/highlands.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+color: $biomes/colors.yml:HIGHLANDS
+
```
Comment (Sancires) [inline biomes/land/mountains-small/temperate/semi-humid/highlands.yml:6]: Again, for future reference, assign the biome its own colors, I recommend going [here](https://minecraft.fandom.com/wiki/Color) and starting from base values from whatever biome you choose. (This is future-proofing for if/when the "vanilla" key is ever dropped.

File: biomes/land/mountains-small/temperate/semi-humid/highlands.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  - threshold: 6
+    palette:
+      - BLOCK:minecraft:stone: 319
+      - << meta.yml:palette-bottom
+
+features:
+  flora:
+    - GRASS
+  trees:
+    - OAK_BUSH_PATCHES
```
Comment (Sancires) [inline biomes/land/mountains-small/temperate/semi-humid/highlands.yml:34]: I think you meant to put spruce bush patches here?

**Review by Sancires — COMMENTED** [review]
Looks good, left notes for future reference. Before I merge: should line 34 of highlands.yml be `SPRUCE_BUSH_PATCHES` ?

**Comment by Vulpicula** [comment]
Nope, this was intentionally set to oak_bush_patches! I made a variant, spruce_bush_patches, to see if it'd work better, but it didn't. I've kept that in though, in case others wish to use it for their biomes.

As for the other comments, thanks for letting me know. I'll do my best to stick with those in the future!

**Pull request merged**